Makefile

```diff
@@ -42,7 +42,7 @@ endif
 DOCKER_UID           = $(shell id -u)
 DOCKER_GID           = $(shell id -g)
 DOCKER_USER          = $(DOCKER_UID):$(DOCKER_GID)
-COMPOSE              = DOCKER_USER=$(DOCKER_USER) DB_HOST=$(DB_HOST) DB_PORT=$(DB_PORT) docker compose
+COMPOSE              = DOCKER_USER=$(DOCKER_USER) DB_HOST=$(DB_HOST) DB_PORT=$(DB_PORT) docker compose -f docker-compose.yml -f docker-compose-$(DB_HOST).yml
```

Comment:
We are also using the `docker compose` command in the file `bin/_config.sh`. helper used in the `bin` directory will probably not work anymore if we don't modify the `_config.sh` file.

Reply:
On `bin/_config.sh` the existing if with `DB_ENGINE` is dead code, because the file `docker/compose/development/mysql/docker-compose.yml` doesn't exist.
From what I see there isn't a proper way to run pytests on mysql locally.

I think I have fixed it.
Now is possible to run:
```bash
DB_HOST=mysql bin/pytest --reuse-db tests/apps/courses/test_templates_course_detail_rdfa.py
```
And only the mysql container is started.

Ref: https://github.com/openfun/richie/blob/7a0bf9ff20b18007529ec9f99fafbd66f266b8de/bin/_config.sh#L12

Reply:
Additionally, I've added support to run the tests on sqlite locally, on a new commit on this PR.

```
DB_HOST=sqlite bin/pytest
```